### PR TITLE
feat: Coverage Dots Everywhere — Waves 0-4 (complete)

### LINKED ISSUE #3902: Coverage dots: add to 6 simple directory tables (Phase 1a)
## Problem

Coverage dots currently exist on only 4 of 15+ directory tables (organizations, people, AI models, legislation). Phase 1a adds coverage dots to the 6 remaining simple custom-table directories: projects, benchmarks, grants, funding-programs, funding-rounds, divisions. Write 5 new per-type scoring functions (project, benchmark, grant, funding-program, funding-round; divisions uses computeGenericCoverage). Add Coverage column to each table. Include score distribution validation: skip column if >80% of entities score identically. Part of Discussion #3901.

## Recommended Model

**Haiku** — well-scoped for this model.

## Acceptance Criteria

- [ ] All 6 tables show coverage dots
- [ ] 5 new scoring functions with tests
- [ ] Score distribution validation passes
- [ ] pnpm build passes

### LINKED ISSUE #3903: Coverage dots: TanStack tables + thin types (Phase 1b)
## Problem

Phase 1b of coverage dots rollout (Discussion #3901). Add coverage dots to the remaining directory tables: approaches, events, research-areas (using computeGenericCoverage), and publications and resources (TanStack React Table pattern using columnHelper.display()). Write publication and resource scorers if score distribution justifies it. Skip columns where >80% of entities score identically.

## Recommended Model

**Haiku** — well-scoped for this model.

## Acceptance Criteria

- [ ] Approaches, events, research-areas tables show coverage dots
- [ ] Publications and resources tables show coverage dots (or explicitly excluded with rationale)
- [ ] pnpm build passes

### PR DESCRIPTION
**Pull request by OAGr**
## Summary

Complete implementation of Coverage Dots across the site (Waves 0-4 of Discussion #3901).

**Wave 0 — Foundation:**
- `CoverageDots` component (1-4 dot ladder, sm/md sizes, tooltip, aria-labels)
- 5 initial scoring functions (org, person, ai-model, legislation, generic)
- Coverage column on 4 tables (organizations, people, AI models, legislation)

**Wave 1 — All scorers:**
- 7 new per-type scoring functions (project, benchmark, grant, funding-program, funding-round, division, publication)
- Generic scorer `filledFieldCount` cap reduced 4→2

**Wave 2 — All remaining table integrations (11 tables):**
- All 15 directory tables now show coverage dots

**Wave 3 — Entity headers + data access:**
- Coverage dots on org, person, and legislation detail page headers
- `entity-coverage.ts` data access layer

**Wave 4 — Search results + dashboard:**
- Coverage dots in entity search results (pre-computed server-side, passed as prop)
- "Data Depth" column added to existing Entities & Pages dashboard (E908)

Closes #3902
Closes #3903

## Test plan

- [x] `pnpm build` passes
- [x] TypeScript check passes (0 errors in changed files)
- [x] 12 scoring functions tested with edge cases
- [x] Review: dead interface fields fixed, input clamping added
- [ ] Visual check: all 15 directory pages render dots
- [ ] Visual check: org/person/legislation headers show dots
- [ ] Visual check: search results show dots for entities
- [ ] Visual check: Entities dashboard shows Data Depth column

### CODE REVIEW

**Comment by OAGr** [comment]
🤖 **PR Patrol** — Attempting fix for: bot-review-major

**Comment by OAGr** [comment]
🤖 **PR Patrol** — Fix attempt complete (73s, 15 max turns, model: sonnet).

**Issues detected**: bot-review-major

**Result**:
zations/page.tsx`** — Changed `computeOrgCoverage({})` to `computeOrgCoverage({ foundedDate })` for local-only orgs so their founded date is counted
4. **`coverage-score.test.ts`** — Switched relative import `../coverage-score` to alias `@/components/coverage/coverage-score` per project conventions

File: .gitignore [hunk 1]
```diff
@@ -39,7 +39,7 @@ apps/web/public/search-index.json
 
 # Test results (Playwright / Vitest artifacts)
 test-results/
-coverage/
+/coverage/
```
[the remark this answers is not in the document]
Comment (OAGr) [reply .gitignore]: Already fixed in ead6c65b4 — added `apps/web/coverage/`, `apps/wiki-server/coverage/`, `apps/discord-bot/coverage/`, and `crux/coverage/` patterns. The tracked component directory `apps/web/src/components/coverage/` is unaffected by these patterns and remains committed.
